website/views/options/profile.jade

```diff
@@ -223,17 +227,14 @@ mixin profileStats
                       span.glyphicon.glyphicon-download
                       |&nbsp;
                       =env.t('distributePoints')
-            +statAllocation
-
+            +statAllocation()
 
+script(id='partials/options.profile.achievements.html', type='text/ng-template')
+  .container-fluid
+    div(class=mobile?'padding':'row')
```

Comment:
Just the ones in this PR. I wouldn't bother going through all the views since the client code is soon to be replaced.